features/admin.connections.v1/components/edit/connection-edit.tsx

```diff
@@ -237,6 +231,7 @@ export const EditConnection: FunctionComponent<EditConnectionPropsInterface> = (
                 data-testid={ `${ testId }-general-settings` }
                 isReadOnly = { isReadOnly }
                 loader={ Loader }
+                hideIdPLogoEditField
```

Comment:
Could we know the intention behind removing the config for showing/hiding the idp logo edit field?

Reply:
The extension config value is redundant and set to true always.

Reply:
We are planning to bring back the idp logo edit capability[1]. Let's keep this config until that is completed. Having the config will be helpful for us to show/hide the field as required, after the feature is onboarded

[1] https://github.com/wso2/identity-apps/pull/6440

Reply:
Ack. Please do the necessary cleanup via that PR.

Reply:
Added back via b2bf84739c245114ce3b54860c004b050bd98ffe